posthog/api/query.py

```diff
@@ -131,9 +136,13 @@ def check_team_api_queries_concurrency(self):
         },
     )
     @monitor(feature=Feature.QUERY, endpoint="query", method="POST")
-    def create(self, request, *args, **kwargs) -> Response:
+    def create(self, request: Request, *args, **kwargs) -> Response:
         data = self.get_model(request.data, QueryRequest)
-
+        with suppress(Exception, ValueError):
```

Comment:
TIL about `suppress` 😄 

nit: I think having `ValueError` is redundant since it inherits from `Exception`